phpBB/language/en/cli.php

```diff
@@ -110,8 +110,10 @@
 
 	'CLI_EXTENSION_DISABLE_FAILURE'		=> 'Could not disable extension %s',
 	'CLI_EXTENSION_DISABLE_SUCCESS'		=> 'Successfully disabled extension %s',
+	'CLI_EXTENSION_DISABLE_DISABLED'	=> 'Extension %s is not enabled',
```

Comment:
No Matt, please take a look at my prior comment: the extension could be available but not disabled, or even not available, so the message would be incorrect stating that it is disabled.  The check is explicitly for `!is_enabled` that is different from `is_disabled`